### DISCUSSION TITLE
Many missing functions in new UI

### DISCUSSION THREAD

**Discussion by TomGalla11** [community/community#6150] [Other Feature Feedback, Questions, & Ideas] [answered]
The new UI isn‘t that good. If you format the changelog like code then no lines are shown. I know many developers who do it like this. I also like it when I see the whole changelog and not just a few lines! The tag button is also missing, the tags are very useful to get all releases on „one“ site. The new design is basically the old UI - (minus) buttons, text,...

Edit: I am on an iPad

**Comment by MylesBorins** [comment, marked as answer]
Thanks for the feedback. We are working on improving the functionality of "read more" to update the content inline of the release rather than take you to a new page. There is still a link for tags at the top of the page on the list view, but we did intentionally remove it from the page of an individual release as it was introducing extra clutter.

Is there anything that is missing for you?
